ktoml-core/src/commonMain/kotlin/com/akuleshov7/ktoml/writers/TomlEmitter.kt

```diff
@@ -180,7 +180,9 @@ public abstract class TomlEmitter(config: TomlOutputConfig) {
             val quotes = if (isLiteral) "'''" else "\"\"\""
 
             emit(quotes)
+                .emitNewLine()
```

Comment:
@NightEule5 can you please review it, if possible?

Reply:
Looks ok